core/vm/celo_contracts.go

```diff
@@ -11,6 +11,10 @@ import (
 	"github.com/holiman/uint256"
 )
 
+var (
+	TRANSFER_PRECOMPILE_ADDRESS = common.HexToAddress("0x00000000000000000000000000000000000000fd")
```

Comment:
Nit, but convention is [mixed caps](https://go.dev/doc/effective_go#mixed-caps), also this is exported and it doesn't look like it needs to be.